pkg/resources/common/common.go

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 2020, MinIO, Inc.
+ *
+ * This code is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License, version 3,
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>
+ *
+ */
+
+package common
+
+import (
+	miniov1 "github.com/minio/minio-operator/pkg/apis/operator.min.io/v1"
+	corev1 "k8s.io/api/core/v1"
+)
+
+func minioPullSecret(mi *miniov1.MinIOInstance) []corev1.LocalObjectReference {
```

Comment:
This function is not exposed who is using this?

Reply:
I removed it and force pushed. should be clean again.